content/en/docs/ops/integrations/spire/index.md

```diff
@@ -27,180 +27,145 @@ The integration is compatible with Istio upgrades.
 
 ## Install SPIRE
 
-### Option 1: Quick start
+Istio recommends you follow SPIRE's installation instructions and general recommendations for installing SPIRE.
 
-Istio provides a basic sample installation to quickly get SPIRE up and running:
+For this guide, the [SPIRE Helm charts](https://artifacthub.io/packages/helm/spiffe/spire) will be used as a simple way to install SPIRE dependencies for Istio and illustrate the configuration necessary to integrate SPIRE and Istio.
 
-{{< text syntax=bash snip_id=install_spire_with_controller_manager >}}
-$ kubectl apply -f @samples/security/spire/spire-quickstart.yaml@
+{{< text syntax=bash snip_id=install_spire_crds >}}
+$ helm upgrade --install -n spire-server spire-crds spire-crds --repo https://spiffe.github.io/helm-charts-hardened/ --create-namespace
 {{< /text >}}
 
-This will deploy SPIRE into your cluster, along with two additional components: the [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi) — used to share the SPIRE Agent's UNIX Domain Socket with the other
-pods throughout the node — and the [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager), a facilitator that performs workload registration and establishes federation relationships
-within Kubernetes. See [Install Istio](#install-istio) to configure Istio and integrate with the SPIFFE CSI Driver.
+{{< text syntax=bash snip_id=install_spire_istio_overrides >}}
+$ helm upgrade --install -n spire-server spire spire --repo https://spiffe.github.io/helm-charts-hardened/ --wait -f - <<EOF
+global:
+  spire:
+    trustDomain: example.org
+EOF
+{{< /text >}}
 
-### Option 2: Configure a custom SPIRE installation
+{{< tip >}}
+This is the minimal subset of changes required to integrate with Istio.  See the [SPIRE Helm chart](https://artifacthub.io/packages/helm/spiffe/spire) documentation for other values you can configure for your installation.
+{{< /tip >}}
 
-See the [SPIRE's Quick start for Kubernetes guide](https://spiffe.io/docs/latest/try/getting-started-k8s/)
-to get started deploying SPIRE into your Kubernetes environment. See [SPIRE CA Integration Prerequisites](#spire-ca-integration-prerequisites)
-for more information on configuring SPIRE to integrate with Istio deployments.
 
-#### SPIRE CA Integration Prerequisites
+This will also install:
 
-To integrate your SPIRE deployment with Istio, configure SPIRE:
+- The [SPIFFE CSI driver](https://github.com/spiffe/spiffe-csi), which is used to mount an Envoy-compatible SDS socket into proxies. Using the SPIFFE CSI driver to mount SDS sockets is strongly recommended by both Istio and SPIRE, as `hostMounts` are a larger security risk and introduce operational hurdles.
 
```

Comment:
```suggestion
```

No extra newline between same list items.

Reply:
yep, this should have been WIP, didn't lint, fixing